data/boosters/lci.yaml

```diff
@@ -2,15 +2,18 @@
 pack:
   cave_fullart_land: 1
   foil_chance:
-  - sfc_common_with_showcase: 9
+  - sfc_common: 9
```

Comment:
Is `sfc_common` here and at line 7 correct? Shouldn't they be still `sfc_common_with_showcase`?

Reply:
There are no single faced, common showcase cards in this set: https://mtg.wtf/card?q=e%3ALCI+is%3Asfc+promo%3Aboosterfun+r%3Ac

Reply:
then possibly the `sfc_common_with_showcase` sheet at line 14 should be `sfc_common`? 
Or maybe everything is correct and I still don't fully understand how you parse these yaml :D if that's the case, sorry for the bother...

Reply:
No yep, that's a good catch I'll fix line 14